src/interface/src/styleguide/treatments-legend/treatments-legend.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input } from '@angular/core';
+import { NgIf, NgFor, NgClass } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { TreatmentTypeIconComponent } from '../treatment-type-icon/treatment-type-icon.component';
+import { SequenceIconComponent } from '../sequence-icon/sequence-icon.component';
+import { ModalComponent } from '../modal/modal.component';
+import {
+  PrescriptionSingleAction,
+  PRESCRIPTIONS,
+} from '../../app/treatments/prescriptions';
+
+/**
+ * Treatment Legend
+ * A component that displays a set of treatments, along with an icon
+ */
+@Component({
+  selector: 'sg-treatments-legend',
+  standalone: true,
+  imports: [
+    MatExpansionModule,
+    MatIconModule,
+    ModalComponent,
+    NgClass,
+    NgIf,
+    NgFor,
+    SequenceIconComponent,
+    TreatmentTypeIconComponent,
+  ],
+  templateUrl: './treatments-legend.component.html',
+  styleUrl: './treatments-legend.component.scss',
+})
+export class TreatmentsLegendComponent {
+  constructor() {}
+
+  openState = false;
+  @Input() singleTreatments: PrescriptionSingleAction[] = [];
+  @Input() sequences: number[] = [1, 2, 3, 4, 5, 6, 7, 8]; // TODO: make dynamic
+
+  loadAvailableTreatments(): void {
+    // TODO: read this from local types instead of lookups?
```

Comment:
@pflopez - when we originally discussed this component, I think we were planning to get our full list of prescription types from the backend. 
Now it seems that `prescriptions.ts` might serve as the FE source of truth. Any opinions on how this component ought to be populated now?

Reply:
i think we could use it directly from prescriptions.ts, however if we go that route, maybe this component needs to live on the app / treatments code and not on the styleguide; unless you can come up with a general "legend" component, and a concrete implementation on app/treatments that does the mapping, which im not sure is worth the effort

Reply:
I say we keep this component on treatments, and maybe invest the effort on the styleguide side for the modal itself which (which you suggested already on another pr :D )

Reply:
I agree, I suspect we'll never be using this anywhere outside of treatments. Styleguide is a convenient place to build/test UI components, but there's no reason this needs to live there.